backendless.d.ts

```diff
@@ -1370,17 +1371,24 @@ declare module Backendless {
 
         addBulkUpsertListener(callback: (list: string[]) => void, onError: (error: RTSubscriptionError) => void): Backendless.EventHandler;
         addBulkUpsertListener(callback: (list: string[]) => void): Backendless.EventHandler;
+        addBulkUpsertListener(whereClause: string, callback: (list: string[]) => void, onError: (error: RTSubscriptionError) => void): Backendless.EventHandler;
```

Comment:
@stanislav-grin please add TSC tests for these changes

Reply:
@Valodya done